dts/bindings/pcie/host/pci-host-ecam-generic.yaml

```diff
@@ -21,3 +21,12 @@ properties:
         As described in IEEE Std 1275-1994, but must provide at least a
         definition of non-prefetchable memory. One or both of prefetchable Memory
         and IO Space may also be provided.
+
+    interrupt-map-mask:
```

Comment:
You add three new properties but these properties are not used in pcie_ecam driver, are they?